docs/questions/sharing/visualizations/pie-or-donut-chart.md

```diff
@@ -1,11 +1,101 @@
+![Pie and sunburst charts](../../images/pie-sunburst-demo.png)
 
-![Inedible donut chart](../../images/donut.png)
+## When to use a pie or sunburst chart
+
+Pie charts can be used to visualize a metric as part of a whole. Sunburst charts can be used to visualize hierarchical or nested data, or display multiple levels of categorization.
+
+Pie and sunburst charts work best when when the number of possible breakout values is small, like accounts by plan. They are useful when yPie charts communicate a general sense of the relative sizes of breakouts, and shouldn't be used when precise comparison is important. If you have more than a few breakouts, like users per country, or you need to communicate exact differences in sizes, it's usually better to use a bar chart.
+
+Pie and sunburst charts should only be used to visualize metrics that add up to 100% (like count or sum of total).
+
+## How to create a pie or sunburst chart
+
+Pie and sunburst charts display a single metric across one or more breakouts. Both pie/donut charts and sunburst charts are available in the "Pie chart" option in the visualization sidebar.
+
+For a simple pie chart, you'll need a query with a single breakout and a numerical metric, for example:
+
+| Type | Sum of Quantity |
+| ---- | --------------- |
+| Cat  | 23              |
+| Bird | 14              |
+| Dog  | 35              |
+
+If your query has multiple metrics, you'll be able to choose the metric for the chart in [chart settings](#pie-and-sunburst-chart-settings).
+
+![Plain pie](../../images/plain-pie.png)
+
+For a sunburst (multi-dimensional pie) chart, you'll need a query with at most three breakouts and a numerical metric. Here's an example of a query result with two breakouts:
+
+| Type | Subtype | Sum of Quantity |
+| ---- | ------- | --------------- |
+| Cat  | Siamese | 4               |
+| Cat  | Persian | 12              |
+| Cat  | Bengal  | 7               |
+| Bird | Crow    | 5               |
+| Bird | Parrot  | 9               |
+| Dog  | Corgi   | 10              |
+| Dog  | Poodle  | 13              |
+| Dog  | Bulldog | 7               |
+| Dog  | Husky   | 5               |
+
+You'll be able to choose which breakouts to assign to inner, middle, or outer ring of the sunburst chart in [chart "Data" settings](#data-settings). If your query has multiple metrics, you'll also be able to choose the metric to display. Here's the sunburst chart for the example table above:
+
+![](../../images/sunburst-two-levels.png)
+
+Metabase will automatically compute percentage of total for each value of the metric value (for example, Metabase will compute that 23 Cats make up 31.9% of all pets), so you don't need to include this information in the query.
```

Comment:
This seems obvious but we had people who didn't realize this (especially for SQL questions)